### DISCUSSION TITLE
milvus 0.10.1  No index is automatically created when the partition reaches the index_file_size limit

### DISCUSSION THREAD

**Discussion by Haohao966** [milvus-io/milvus#3688] [Q&A and General discussion]
There are two partitions in a collection,One partition has 2.1 million and the other has 1.1 million,The vector dimension is 256,float32

index_file_size :2048M

The HNSW index has been previously indexed in parts of the collection
However, the current performance query is very slow, which should be caused by the fact that new inserts are not indexed

**Comment by shiyu22** [comment]
You can get the metadata through a Sqlite or Mysql database, see https://www.milvus.io/cn/blogs/2019-12-27-meta-table.md. You can check which files are not yet indexed and the size of each segment.

Milvus will build index automatically when it reaches index_file_size, and you can also manually call create_index() to index the rest of the data.
